apidef/oas/default_test.go

```diff
@@ -1125,6 +1128,7 @@ func TestOAS_importAuthentication(t *testing.T) {
 			}
 
 			assert.Equal(t, expectedSecuritySchemes, authentication.SecuritySchemes)
+			assert.Equal(t, apidef.AuthTypeEnum(""), authentication.BaseIdentityProvider)
```

Comment:
Could we declare a readable zero type here, e.g. `apidef.AuthTypeNone`? Also repeated 2x below. Also avoids allocations on use.

Reply:
👍🏻